### DISCUSSION TITLE
jira issue view not showing comments

### DISCUSSION THREAD

**Discussion by gennaro-tedesco** [ankitpokhrel/jira-cli#180] [Ideas] [answered]
Apologies for opening so many issues (I am just using the tool a lot!). I have noticed that `jira issue view <ticker-nr>` does not by default parse comments text. Is this intended behaviour or did I miss the documentation explaining how to visualise comments?

Likewise using `jira issue comment` lets you submit a comment but does not display the comments section either.

**Comment by ankitpokhrel** [comment]
Hi @gennaro-tedesco, this feature is on the to-do list and yet to be implemented. I will work on it as soon as I get some time.

Glad to know that the tool is helpful for you. Also, bug reports and PRs are welcome :)

**Comment by gennaro-tedesco** [comment]
Thank you, I did start having a look at the code and hopefully I will be able to help with it a little! :)

**Comment by bluejellybean** [comment]
I'm just starting to check out this codebase (and LOVING the entire project!), but my go skills are a bit lacking. Has there been any progress or movement on this card issue??

**Comment by ankitpokhrel** [comment]
@gennaro-tedesco @bluejellybean The tool now displays latest comment when viewing the issue. 
![Screen Shot 2021-12-05 at 7 50 06 PM](https://user-images.githubusercontent.com/2364546/144763192-1a1548dd-eff6-4419-82fc-207c6dae77a5.png)
This feature will be available in the next release. For now, if you wish to use this feature, you can [compile from source](https://github.com/ankitpokhrel/jira-cli#development) or if you are using homebrew you can build binary from `HEAD` (requires `make`, `git` and `go`)

```sh
brew uninstall jira-cli
brew install jira-cli --HEAD
``` 

An option to view `n` number of comments will hopefully be done soon.

**Reply by gennaro-tedesco** [reply]
Wooow, awesome, I am going to test it soon! :)

**Comment by ankitpokhrel** [comment, marked as answer]
This feature is already available in [the latest release v0.3.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v0.3.0)

**Comment by mslinn** [comment]
There should be an option to show all comments. Currently, one must specify a large number instead, like this:

```
jira issue view ISSUE-1 --comments 9999
```

One of the following suggestions would be better:
```
jira issue view ISSUE-1 --comments all
```
```
jira issue view ISSUE-1 --allcomments
```
